src/styles/helpers/color/getBrightness.js

```diff
@@ -0,0 +1,17 @@
+import isValidColor from './isValidColor';
+import decomposeColor from './decomposeColor';
+import hslToRgb from './hslToRgb';
+
+/* http://www.w3.org/TR/AERT#color-contrast */
+
+/* eslint-disable no-param-reassign */
+export default function getBrightness(color) {
+    if (isValidColor(color)) {
```

Comment:
the validation of the color should be made only one time at the top level normalization, at this moment the color will be always valid, it will also avoid  returning undefined .. which is weird :)